includes/amp-enhancements/class-newspack-cookie-notice-amp-markup.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Overrides Cookie_Notice_AMP_Markup to remove ajax call.
+ *
+ * @package Newspack
+ */
+
+use Felix_Arntz\WP_GDPR_Cookie_Notice\Cookie_Notice\Cookie_Notice_AMP_Markup;
+
+namespace Newspack;
```

Comment:
The namespace declaration needs to go before the `use` statement, otherwise it triggers a fatal for me:

```
PHP Fatal error:  Uncaught Error: Class 'Cookie_Notice_AMP_Markup' not found in /Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/includes/amp-enhancements/class-newspack-cookie-notice-amp-markup.php:17
```

Reply:
Good catch. Fixed in https://github.com/Automattic/newspack-plugin/pull/622/commits/c07874d2b9f226b174a69d83760e04a68a42bee3. I wasn't able to reproduce that issue. Just curious, what PHP version are you using?

Reply:
My environment is on PHP 7.4.1.